external/fv3fit/tests/training/test_main.py

```diff
@@ -506,15 +506,7 @@ def cli_main(args: argparse.Namespace):
         ),
         pytest.param("convolutional", {}, ["dQ1", "dQ2"], id="convolutional"),
         pytest.param("precipitative", {}, ["dQ1", "dQ2"], id="precipitative"),
-        pytest.param(
-            "dense", {"save_model_checkpoints": False}, ["dQ1", "dQ2"], id="dense"
-        ),
-        pytest.param(
-            "dense",
-            {"save_model_checkpoints": True},
-            ["dQ1", "dQ2"],
-            id="dense_with_checkpoints",
-        ),
+        pytest.param("dense", {}, ["dQ1", "dQ2"], id="dense"),
```

Comment:
I think I'm OK approving without tests for this new feature, since the registration pattern is simple and testing this could be very complex. Consider whether there are any tests you'd like to add.